protocol/resources/describedby-unique.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Server may link to one description resource 
+
+  Background: Set up clients and paths
+    * def testContainer = rootTestContainer.createContainer()
+    
+  Scenario: Server sees at most one Link to description resource from container
+    Given url testContainer.url
+    When method GET
+    * def links = parseLinkHeaders(responseHeaders)
+    * def test = links.filter(el => el.rel === 'describedBy')
+    And assert (test.length <= 1)
```

Comment:
```suggestion
    Then assert (test.length <= 1)
```
Just a style thing but I think it helps readability of the test: Given x When y Then z